spec/controllers/admin_posts_controller_spec.rb

```diff
@@ -66,7 +66,7 @@
       end
     end
 
-    context "when admin does not have correct authorization" do
+    context "when admin has correct roles" do
```

Comment:
Duplicate context descriptions is [actually a violation](https://docs.rubocop.org/rubocop-rspec/cops_rspec.html#rspecrepeateddescription), which is nice!